distributed/scheduler.py

```diff
@@ -2070,7 +2102,7 @@ def _transition(self, key, finish: str, *args, **kwargs):
 
                 start = "released"
             else:
-                raise RuntimeError("Impossible transition from %r to %r" % start_finish)
+                raise RuntimeError(f"Impossible transition from {start} to {finish}")
```

Comment:
Would change the logic a little, but would make sense to me to have `_transition_dispatch` raise this error if none of the cases match.